lib/BackupPC/Lib.pm

```diff
@@ -96,10 +96,7 @@ sub new
         };
     }
 
-    my $bpc = bless {
-	%$paths,
-        Version => '4.3.3',
-    }, $class;
+    my $bpc = bless {%$paths, Version => '4.3.3',}, $class;
```

Comment:
```suggestion
    my $bpc = bless {%$paths, Version => '4.3.3'}, $class;
```
This extra comma breaks BackupPC on Travis. I have no idea why. I tested it on FreeBSD 12.1 Perl v5.30.2 and it worked even with comma.

Reply:
Seems like @craigbarratt fixed this with f3ac5a25a9d68fe467c6531a9931bb3cfc8b511a